flink-ml-core/src/main/java/org/apache/flink/ml/param/ParamValidators.java

```diff
@@ -95,4 +95,9 @@ public boolean validate(T value) {
             }
         };
     }
+
+    // Check if the parameter value array is not empty array.
+    public static <T> ParamValidator<T> nonEmptyArray() {
```

Comment:
Could you test this validator in `StageTest::testValidators(...)`?

Reply:
OK. I'll add tests.